README.md

```diff
@@ -1,74 +1,23 @@
-    * Debian Jessie
-    * Fedora 25
-  - Windows 10 Pro
+Having trouble? See our [support options below](#support). Additionally, you may have trouble if [another local development tool is already using port 80 or 443](https://ddev.readthedocs.io/en/latest/#using-ddev-with-other-development-environments).
 
-We are open to expanding this list to include additional OSs as well as improve our existing support for the ones listed above. Please [let us know](https://github.com/drud/ddev/issues/new) if you hit an issue!
+## Current Feature List
 
-### Using ddev with other development environments
-ddev requires ports 80 and 3306 to be available for use on your system when sites are running. If you are using another local development environment alongside ddev, please ensure the other environment is turned off or otherwise not using ports 80 and 3306.
-
-If you need to use another environment after using ddev, simply ensure all of your ddev sites are stopped or removed. ddev only uses system ports when at least one site is running.
-
-## Quickstart
-- [Wordpress](https://ddev.readthedocs.io/en/latest/users/cli-usage#wordpress-quickstart)
-- [Drupal 7](https://ddev.readthedocs.io/en/latest/users/cli-usage#drupal-7-quickstart)
-- [Drupal 8](https://ddev.readthedocs.io/en/latest/users/cli-usage#drupal-8-quickstart)
-
-## Installation
-### Homebrew - macOS
-
-For macOS users, we recommend downloading and installing ddev via [homebrew](https://brew.sh/):
-```
-brew tap drud/ddev && brew install ddev
-```
-Later, to upgrade to a newer version of ddev, simply run:
-```
-brew upgrade ddev
-```
-  
-### Installation Script - MacOS or Linux
-
-Linux and macOS end-users can use this line of code to your terminal to download, verify, and install ddev using our [install script](https://github.com/drud/ddev/blob/master/install_ddev.sh):
-```
-curl https://raw.githubusercontent.com/drud/ddev/master/install_ddev.sh | bash
-```
-
-### Manual Installation - Linux and macOS
-You can also easily perform the installation manually if preferred:
-- Download and extract the latest [ddev release](https://github.com/drud/ddev/releases) for your architecture.
-- Make ddev executable: `chmod ugo+x ddev`
-- Move ddev to /usr/local/bin: `mv ddev /usr/local/bin/` (may require sudo), or another directory in your `$PATH` as preferred.
-- Run `ddev` to test your installation. You should see ddev's command usage output.
-
-### Manual Installation - Windows
-- Download and extract the latest [ddev release](https://github.com/drud/ddev/releases) for Windows.
-- Copy `ddev.exe` into `%HOMEPATH%\AppData\Local\Microsoft\WindowsApps`, or otherwise add `ddev.exe` to a folder defined in your `PATH`
-- Run `ddev` from a Command Prompt or PowerShell to test your installation. You should see ddev's command usage output.
-
----
-
-## Usage
 ```
 ➜  ddev
 This Command Line Interface (CLI) gives you the ability to interact with the ddev to create a local development environment.
```

Comment:
s/with the ddev/with ddev/
Better yet, why not just simplify it more.
"The ddev CLI lets you create a local web development environment."

Reply:
+1